src/main/com/algos/src/maths/AbsoluteMax.java

```diff
@@ -13,12 +13,12 @@ public static int getMaxValue(int... numbers) {
         if (numbers == null || numbers.length == 0) {
             throw new IllegalArgumentException("Numbers array cannot be empty or null");
         }
-        int absMax = numbers[0];
+        int absMax = 0;  
         for (int i = 1; i < numbers.length; i++) {
-            if (Math.abs(numbers[i]) > Math.abs(absMax)) {
+            if (numbers[i] > absMax) {
```

Comment:
**Issue**: The comparison logic has been changed from comparing the absolute values to comparing the actual values. This change defeats the purpose of finding the maximum absolute value, as negative numbers with large magnitudes might be ignored. <br> **Fix**: Restore the comparison of absolute values to ensure the function correctly identifies the maximum absolute value. <br> **Code Suggestion**: 
 ```
 -            if (numbers[i] > absMax) {
 +            if (Math.abs(numbers[i]) > Math.abs(absMax)) {
 ```